mudlet-mapper.xml

```diff
@@ -3573,9 +3602,11 @@ mmp.echo("We're connected to StickMUD.")</script>
 					<colorTriggerBgColor>#000000</colorTriggerBgColor>
 					<regexCodeList>
 						<string>The door is locked.</string>
+						<string>The (walnut|pine|oak|iron|reinforced) door is locked\.</string>
```

Comment:
```suggestion
						<string>The (?:walnut|pine|oak|iron|reinforced) door is locked\.</string>
```

Reply:
don't mind making it noncapturing, but curious what is gained by doing so?

Reply:
Just a habit and afaik it's a teeny bit quicker. Also signals the intention better